tests/testthat/test-step-call-pivot_wider.R

```diff
@@ -0,0 +1,240 @@
+test_that("can pivot all cols to wide", {
```

Comment:
It would be useful to add a collection of snapshot tests that capture the generated expressions for key tasks. That makes it possible to review the translations, not just the results.